docs/user/user.yaml

```diff
@@ -648,3 +604,84 @@ paths:
                 status: 404
                 code: DOCUMENT_NOT_FOUND
                 message: User with the specified id was not found.
+  /users/{id}/offers:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - Users
+      summary: Find offers for a user with the specified ID
+      description: Finds and returns all offers for a user with the specified ID.
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/offers'
+              example:
+                count: 2
+                offers:
+                  - _id: 63ebc6fbd2f34037d0aba791
+                    price: 111
+                    proficiencyLevel: [Beginner]
+                    title: offer title
+                    description: offer description
+                    languages: [English]
+                    authorRole: tutor
+                    authorFirstName: Amanda
+                    authorLastName: Smith
+                    authorAvgRating: 4
+                    author:
+                      _id: 63e63bb04d1bf3bea00e3d88
+                      totalReviews:
+                        student: 0
+                        tutor: 0
+                      FAQ: {
```

Comment:
FAQ here is an array of objects, not object

Reply:
fixed